src/sync/sync.go

```diff
@@ -64,6 +65,12 @@ func NewSyncServer(client *db.Client) *SyncServer {
 	return &SyncServer{r: createRouting(client)}
 }
 
+type applicationParams struct {
+	Username      string
+	Application   string
+	Authorization string
```

Comment:
Make these properties lowercase in the JSON:

`Username string `json:"username"``